packages/dashmate/src/core/quorum/waitForDKGWindowPass.js

```diff
@@ -0,0 +1,31 @@
+import { MIN_BLOCKS_BEFORE_DKG } from '../../constants.js';
+import wait from '../../util/wait.js';
+
+/**
+ * @param {RpcClient} rpcClient
+ * @return {Promise<void>}
+ */
+export default async function waitForDKGWindowPass(rpcClient) {
+  const { result: startBlockchainInfo } = await rpcClient.getBlockchainInfo();
+  const { blocks: startBlock } = startBlockchainInfo;
+
+  const { result: startNextDKGInfo } = await rpcClient.quorum('dkginfo');
+  const { next_dkg: startNextDKG } = startNextDKGInfo;
+
+  let isInDKG = true;
+
+  while (isInDKG) {
+    await wait(1000);
```

Comment:
Let's do 10 seconds? Another option is to subscribe to ZMQ to receive notifications about new blocks